index-test.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+require('./src/ImageSequencer');
+sequencer = ImageSequencer({ ui: true });
+var saveSequence = require('./src/cli/saveSequence.js');
+var test = require('tape');
+var { Command } = require('commander');
+
+
+test('testing save sequence function', function (t) {
+  try {
+    var program = new Command();
+    program
+      .option('--save-sequence [string]', 'Name space separated with Stringified sequence');
+    
+    program.parse(['node', 'test', '--save-sequence', '"invert brightness"']);
+
+    if (program.saveSequence)
+      saveSequence(program, sequencer);
+    t.true(1, 'creation success');
+
+  } catch (error) {
+    t.true(!error, 'creation fail');
+  }
+  try {
+    var program = new Command();
+    program
+      .option('--save-sequence [string]', 'Name space separated with Stringified sequence');
+    
+    program.parse(['node', 'test', '--save-sequence']);
```

Comment:
Oh cool, is this the key function, `.parse()`? via Commander? 

This looks great. Could we rename the file something like `cli-test.js` or `index-cli-test.js` so it's clear? And, perhspas should we place it in the `test` directory?

Thanks so much! This is super awesome! And very readable!

Reply:
Yup we pass can pass the user arguments through it. More info [here](https://github.com/tj/commander.js#parse-and-parseasync)

I named it `index-test` since currently it's an standalone executable command for our program.
So the convention is `./index.js <name of command>` tries to find `index-<name of command>` and execute it. More info [here](https://github.com/tj/commander.js#stand-alone-executable-subcommands)

I am still not familiar with the how exactly the current tests are running, I will try to move it to our `test` directory once I get more familiar with it since it doesnt have a dependency on `index.ts` as such so I believe that's possible.

Reply:
ahhh, i see. Yes, let's try to put it into test, but, if the convention doesn't accommodate subdirectories, we can just put a really clear explanatory comment at the top of the file. How does that sound? Thanks!!